core/cloudflow-akka/src/main/resources/jdbcpersistence.conf

```diff
@@ -0,0 +1,41 @@
+akka {
```

Comment:
When I see this, I think, the user likely wants to modify this through config files using `kubectl cloudflow configure` et al. and maybe some of this should go to the cli... reading further..

Reply:
This can be yet another pr - reconfigure application conf through CLI. The way I was trying to implement this is to default to all recommended values, to make the initial implementation simple. Once they have it working, they can optimize configuration, but underlying theme - make it simple to start